space-cli/cmd/modules/ingress/delete.go

```diff
@@ -0,0 +1,52 @@
+package ingress
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/spaceuptech/space-cloud/space-cli/cmd/model"
+	"github.com/spaceuptech/space-cloud/space-cli/cmd/utils/filter"
+	"github.com/spaceuptech/space-cloud/space-cli/cmd/utils/transport"
+)
+
+func deleteIngressGlobalConfig(project string) error {
+
+	// Delete the ingress global config from the server
+	url := fmt.Sprintf("/v1/config/projects/%s/routing/ingress/global", project)
+
+	payload := new(model.Response)
+	if err := transport.Client.MakeHTTPRequest(http.MethodPost, url, map[string]string{}, payload); err != nil {
```

Comment:
```suggestion
	if err := transport.Client.MakeHTTPRequest(http.MethodPost, url, map[string]string{}, new(model.Response)); err != nil {
```
Do this everywhere, where you are not using the payload